x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -57,6 +61,12 @@ thereby limiting the access scope for API keys.
 The structure of role descriptor is the same as the request for create role API.
 For more details, see <<security-api-put-role, create or update roles API>>.
 
+NOTE: Due to the way in which this permission intersection is calculated, it is not
+possible to create an API key that is a child of another API key, unless the derived
+key is created without any privileges. In this case, you must explicitly specify a
+role descriptor with no privileges, and the derived API key can be used for
```

Comment:
Not mandatory, but I think this could be split into two sentences.
```suggestion
role descriptor with no privileges. The derived API key can be used for
```